packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -2,6 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// @docImport 'dart:typed_data';
```

Comment:
Genuine question: Is there anything to take-away from a file like this? So many necessary doc imports! Is it a smell? Is it just a sign of a well-documented library with lots of examples from not-imported code? I honestly didn't think this feature was needed _this_ much!

Reply:
This is a low-level glue file that connects the engine with the framework. It contains lots of hooks for a variety of mechanisms in higher levels. So, all the docs are "this is the hook to power the [foo] thing". Therefore, I am not surprised that this particular file has all these doc imports.